From 32678a29e6e1c41038537a4f2297a737fe9bbd4a Mon Sep 17 00:00:00 2001
From: Jason Hung <jhung@globalscaletechnologies.com>
Date: Wed, 9 Jul 2025 14:24:58 +0800
Subject: [PATCH] configs: add missing CONFIG_SYS_BOOTM_LEN=0x5000000

Without this setting, bootm may fail with decompression errors such
as "inflate returned -5" when handling compressed images exceeding
the default allocation size
 - CONFIG_SYS_BOOTM_LEN=0x5000000
---
 configs/globalscale_mt510ss_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/globalscale_mt510ss_evk_defconfig b/configs/globalscale_mt510ss_evk_defconfig
index 662aee3e84..5b1656b101 100644
--- a/configs/globalscale_mt510ss_evk_defconfig
+++ b/configs/globalscale_mt510ss_evk_defconfig
@@ -25,6 +25,7 @@ CONFIG_DEFAULT_FDT_FILE="gti-mt510ss-evk"
 # CONFIG_DISPLAY_BOARDINFO is not set
 # CONFIG_CMD_CONSOLE is not set
 # CONFIG_CMD_BOOTD is not set
+CONFIG_SYS_BOOTM_LEN=0x5000000
 CONFIG_CMD_BOOTEFI_SELFTEST=y
 # CONFIG_CMD_ELF is not set
 # CONFIG_CMD_GO is not set
